### PR TITLE
Add ksoap/1.8.0

### PR DESCRIPTION
**Pull request by renatofilho**
ksoap/1.8.0

### CODE REVIEW

**Comment by uilianries** [comment]
Oi @renatofilho!

Could you please put your name in #4 ? Some admin will put your user in our EAP. Regards!

File: recipes/kdsoap/all/conanfile.py [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+    author = "Klaralvdalens Datakonsult AB (KDAB) info@kdab.com"
+    exports_sources = ["CMakeLists.txt"]
+    url = "https://github.com/KDAB/KDSoap.git"
+    description = "KD Soap is a Qt-based client-side and server-side SOAP component."
+    settings = "os", "arch", "compiler", "build_type"
+    generators = "cmake"
+    _source_subfolder = "source_subfolder"
+
+    def requirements(self):
+        self.requires("qt/5.13.2@bincrafters/stable")
```
Comment (uilianries) [inline recipes/kdsoap/all/conanfile.py]: unfortunately, Bincrafters and Conan Community package are not allowed. We need to have Qt in CCI first.

File: recipes/kdsoap/all/CMakeLists.txt [hunk 2]
```diff
@@ -0,0 +1,8 @@
+cmake_minimum_required(VERSION 2.8.11)
+project(cmake_wrapper)
+
+message(STATUS "CMake Conan Wrapper")
```
Comment (Croydon) [inline recipes/kdsoap/all/CMakeLists.txt]:
```suggestion
```

File: recipes/kdsoap/all/conanfile.py (hunk 1 above)
Comment (Croydon) [reply recipes/kdsoap/all/conanfile.py]: Until Qt is in CCI it can take some long time. Sorry about that 😅

**Comment by danimtb** [comment]
Unfortunately, it will take quite some time till the qt package can be added to the index. For that time the recipe would need to address additional changes, so better to open a new PR then. Thanks!

**Pull request closed without merging**